project_task_project_required/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Project: require Project on Task',
+    'summary': """
+        Set project on task as a mandatory field""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author':
+        'Brainbean Apps, '
+        'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/project',
+    'depends': [
+        'project',
```

Comment:
```suggestion
        'project',
        'hr_timesheet',
```
Error when I not install hr_timesheet module.

Reply:
Fixed origin of the issue